daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
@stevvooe I am aware of the upcoming internal proposal but it is going to be more than a convention. We will delay following that until it becomes a reality. For now, we will just remove `pkg`